### ISSUE TITLE
Can you help me to start this project directly on esp12e?

### ISSUE THREAD

**Issue by brutevinch** [MrPsi/NodeMCU-Wixel#10]
1) Sucsessfully wrote nodemcu firmware.
2) Upload all lua files to meu
3) After restart i Got IP and can ping this address.
Thats all (( 50005 doesn't work, in terminal there is no other debug information.

**Comment by MrPsi** [comment]
If you want to get debug prints you can set "allowPrint = true" in modStart.lua. Then you will see debug information in ESPlorer, or any other terminal you choose to connect to the NodeMCU with. I think that is the best way of finding out what is happening.

**Comment by brutevinch** [comment]
All i see with  "allowPrint = true"  is: 
![default](https://cloud.githubusercontent.com/assets/12526597/15945010/bc5e1f88-2ea9-11e6-8efc-18b728b2a0f5.PNG)
But i try to start without connected wixel. Only ESP8266 is it matter?

**Comment by danpowell88** [comment]
It will print when receiving data from the wixel i think, possibly also when a client connection is created, although I may have just added that myself locally for testing (cant remember)

**Comment by brutevinch** [comment]
@MrPsi @thecubical Have you any idea how to debug my current state?

**Comment by MrPsi** [comment]
You should see more text like "Connecting..." while connecting to the wifi. But all text except the text you see is printed using the modPrintText module. The modPrintText module switches between the primary and secondary serial pins while writing. Because the primary pins is used to print text, and the secondary pins is used for reading text from the wixel. I'm not familiar with the pinout for esp12e, but it might be that it do not work the same regarding the primary and secondary serial pins. To be able to debug you could edit modPrintText and comment out all code in the modPrintText.print method except the "print(text)" at the end. To not switch to the secondary serial pins for reading the wixel you can also edit modUart and comment out all code in modUart.start.

**Comment by runeron** [comment]
Probably not related, but I will mention it just in case (using NodeMCU, not just the esp)..

I had problems getting the wixel to work, untill I changed the Baud rate from 9600 to 115200 (yeah, strange). Once I changed it, I got a lot more text in the display, and everything magically started to work.

w/9600: I could upload files (would timeout sometimes), and I would see the same text as you have linked above ("starting wixel in 10 seconds"), and I even got an IP from time to time, but could not connect in monitor.
